### PR TITLE
conda-build v3.18.1

### PR DESCRIPTION
**Pull request by msarahan**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

**Comment by msarahan** [comment]
aarch64 and ppc are expected to fail, due to lack of libarchive and/or LIEF.  LIEF isn't strictly necessary, and we could perhaps explore a way of making that a soft requirement (use if present, warn and skip that step if not present)

**Comment by isuruf** [comment]
> aarch64 and ppc are expected to fail

Is this a regression for conda-build 3.18.0?

**Comment by msarahan** [comment]
not that I know of?  Maybe I'm just behind on what you guys have achieved on those platforms.  I'm not aware of LIEF or libarchive on power or aarch64, but maybe you know more?

**Comment by isuruf** [comment]
py-lief and liblief are available for aarch64. https://github.com/conda/conda-build/pull/3446 made lief optional for ppc64le.

**Comment by isuruf** [comment]
aarch64 build was failing because `conda-package-handling` was not built for that platform. It's built now, but not mirrored in CDN for some reason.

**Comment by msarahan** [comment]
going to cut another release for fixing the win build stuff.

**Comment by msarahan** [comment]
whoops, wrong PR, sorry.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by scopatz** [comment]
Is this OK to merge?

**Comment by isuruf** [comment]
I still can't figure out why CircleCI and Travis-CI are not removed with a rerender

**Comment by dhirschfeld** [comment]
Any reason not to merge this? I'm keen to give the new release a go!

**Comment by scopatz** [comment]
Merging under advisement of @msarahan

**Comment by msarahan** [comment]
oh dear.  Famous last words.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,7 +38,6 @@ requirements:
     - filelock
     - futures            # [py27]
     - jinja2
-    - python-libarchive-c
```
Comment (isuruf) [inline recipe/meta.yaml]: Why was this removed?
